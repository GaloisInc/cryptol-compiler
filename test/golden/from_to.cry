main = [ 2^^64 - 1 .. 2^^64 + 5 : Integer]


