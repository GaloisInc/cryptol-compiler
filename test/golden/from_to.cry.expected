[0xffffffffffffffff, 0x10000000000000000, 0x10000000000000001, 0x10000000000000002, 0x10000000000000003, 0x10000000000000004, 0x10000000000000005]

